app/assets/stylesheets/tables-report.css.scss

```
@@ -15,3 +25,20 @@
     }
   }
 }
+
+.usa-table {
```

Even though the scope of the how the styles are applied is limited, I think it could be good to create an additional class to distinguish these tables as nonstandard.

e.g.

```suggestion
.usa-table.report-table {
```